project/Dependencies.scala

```diff
@@ -62,14 +63,14 @@ object Dependencies {
   val apiDeps = Seq(libraryDependencies ++= baseDeps ++ Seq(
     akkaActor,
     akkaTestKit,
+    akkaHttp(""),
+    akkaHttp("-spray-json"),
+    akkaHttpCirce,
+    akkaHttpTestKit,
     cats % "test",
     circe("-core"),
-    circe("-spray"),
     newRelic,
-    spray("-can"),
-    spray("-routing-shapeless2"),
-    sprayJson,
-    sprayTestKit
+    sprayJson
```

Comment:
We use `circe` to implement the `services`, for encoding messages, and we used `spray-json` to implement the `api`because we were more familiar with it; but we may consider migrating `spray-json` encoders and decoders to `circe`, in due time.